systemd/drone.env

```diff
@@ -0,0 +1,4 @@
+DRONE_HOST=FILLME
+DRONE_GITHUB_CLIENT=FILLME
```

Comment:
Empty value is preferable, as then value will not be set and dependent scripts have a chance to raise an appropriate error